code/datums/jobs.dm

```diff
@@ -144,6 +144,14 @@
 		..()
 		src.access = get_all_accesses()
 
+	special_setup(var/mob/living/carbon/human/M)
+		..()
+		if (!M)
+			return
+		if(prob(5*ASS_JAM))
```

Comment:
Better to wrap things in #if/#else/#endif than to use the define directly in the code, for ease of searching and clearly telling what things do